challenges/support/README.md

```diff
@@ -0,0 +1,66 @@
+# Summary
+
+The Solutions and Support Engineer sysadmin challenge is a simulated troubleshooting session where several faults have been created in a
+sample environment requiring the candidate to investigate and solve the problems. 
+
+
+> **Note:** For SREs/Devops without a backend development background, the on-call sysadmin challenges are used, but instead of remote hands, shell access to the nodes will be provided by provisioning a public ssh key.
```

Comment:
What does this mean? What are the "on-call sysadmin challenges"? Can we link to them?

Reply:
this is a leftover, I will remove it